environment/commands/clean.js

```diff
@@ -17,13 +17,14 @@ class CleanCommand {
             });
         });
         try {
-            execSync('python3 ~/.rainbowup/nearup/nearup stop');
+            execSync('python3 ~/.rainbowup/nearup/main.py stop');
         } catch (err) {
-            console.log(`Error stopping nerup ${err}`);
+            console.log(`Error stopping nearup ${err}`);
         }
-        console.log('Cleaning ~/.rainbowup , ~/.nearup , and ~/.near directories...');
-        execSync('rm -rf ~/.rainbowup && rm -rf ~/.nearup && rm -rf ~/.near');
+        console.log('Cleaning ~/.rainbowup directory...');
+        execSync('rm -rf ~/.rainbowup');
```

Comment:
imo `~/.near` is dangerous, also since rainbow bridge doesn't use `~/.nearup` it also should not clean up it.